### PR TITLE
Change begin/end date and start/end time

### PR DESCRIPTION
**Pull request by amandine-sahl**

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -5,6 +5,12 @@ CHANGELOG
 2.87.2+dev (XXXX-XX-XX)
 -----------------------
 
+
+**Minor improvements**
+- Set begin_date, end_date not null for touristic event
+- Rename metting_time to start_time
```
Comment (camillemonchicourt) [inline docs/changelog.rst:11]: - metting_time > meeting_time
- You should mention in which module (Touristic events)

File: docs/changelog.rst [hunk 2]
```diff
@@ -8,6 +8,8 @@ CHANGELOG
 
 **Minor improvements**
 - Set begin_date, end_date not null for touristic event
+- Rename metting_time to start_time
```
Comment (camillemonchicourt) [inline docs/changelog.rst:11]: We should also mention it as BREAKING CHANGES in API V2

File: docs/changelog.rst [hunk 3]
```diff
@@ -5,6 +5,10 @@ CHANGELOG
 2.87.2+dev (XXXX-XX-XX)
 -----------------------
 
+
+**Minor improvements**
+- Set begin_date, end_date not null for touristic event
```
Comment (Chatewgne) [inline docs/changelog.rst:10]: Remove `end_date` here

File: geotrek/tourism/migrations/0027_auto_20220927_0814.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tourism', '0026_auto_20220907_1400'),
+    ]
+
+    operations = [
+        migrations.RunPython(update_begin_end_date, reverse_code=migrations.RunPython.noop)
```
Comment (Chatewgne) [inline geotrek/tourism/migrations/0027_auto_20220927_0814.py:26]: Rename `update_begin_end_date` to `update_begin_date`

File: geotrek/tourism/tests/test_views.py [hunk 5]
```diff
@@ -513,19 +513,17 @@ def test_touristic_events_without_enddate_filter(self):
     def test_touristic_events_with_enddate_filter(self):
         """
         Relative date: 2020-01-01
-        5 events with no end date
         5 events with end date after relative date
         7 events with end date before relative date
                  ->  only events with no end or end in after relative date must be included
```
[the remark this answers is not in the document]
Comment (amandine-sahl) [reply geotrek/tourism/tests/test_views.py:519]: Dans ce cas là les éléments n'ayant pas de date de fin sont présents

File: geotrek/api/v2/serializers.py [hunk 6]
```diff
@@ -454,6 +454,9 @@ def get_type(self, obj):
                 return obj_type.pk
             return None
 
+        def get_end_date(self, obj):
+            return obj.end_date or obj.begin_date
```
Comment (Chatewgne) [inline geotrek/api/v2/serializers.py:458]:
```suggestion
            return obj.end_date if obj.end_date else obj.begin_date
```

File: geotrek/api/v2/filters.py [hunk 7]
```diff
@@ -500,10 +500,14 @@ def filter_queryset(self, request, queryset, view):
...
                 dates_after = datetime.strptime(dates_after, "%Y-%m-%d").date()
+                qs = qs.filter(Q(end_date__gte=dates_after))
             else:
                 # Filter out past events by default
                 dates_after = date.today()
-            qs = qs.filter(Q(end_date__gte=dates_after))
+                qs = qs.filter(
+                    Q(end_date__gte=dates_after) | 
+                    Q(end_date__isnull=True) & Q(begin_date__gte=dates_after)
+                )
```
Comment (Chatewgne) [inline geotrek/api/v2/filters.py:510]: Add tests
- Events with no end_date are returned in APIv2 with begin_date as value

File: docs/changelog.rst [hunk 8]
```diff
@@ -5,6 +5,10 @@ CHANGELOG
 2.87.2+dev (XXXX-XX-XX)
 -----------------------
 
+
+**Minor improvements**
+- Set begin_date not null for touristic events
+
```
Comment (Chatewgne) [inline docs/changelog.rst:11]: Add 

Bug fix : 
TouristicEvents with no end dates are not returned in APIv2

Comment (camillemonchicourt) [reply docs/changelog.rst:11]: With a reference to the dedicated issue (#3127)

**Pull request merged**